model.md

```diff
@@ -0,0 +1,4 @@
+The *verification* problem can be solved by cryptographic signing (implemented in IPFS)  
+The *authority* problem can be solved by webs of trust (to be implemented in Data Together app)  
+The *discovery* problem can be solved by ??? (archivers has a “coverage” service that might be a model, but ultimately this should be decentralized, right?)  
+The *access* problem feels like a complex one, actually.
```

Comment:
any more details on that?

Reply:
haha I think these were your own words from loooong ago @titaniumbones 😂